From 6cc26ef0b749f56f184faacde210c6674ab83cf4 Mon Sep 17 00:00:00 2001
From: Ronny Chevalier <chevalier.ronny@gmail.com>
Date: Tue, 7 Apr 2015 19:35:57 +0200
Subject: [PATCH] zsh-completion: add missing -M completion for journalctl

(cherry picked from commit b178d279d92fdf002b18dd2f06f2353af14d0a6e)

Cherry-picked from: 6ff6bbf
Resolves: #1222517
---
 shell-completion/zsh/_journalctl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/shell-completion/zsh/_journalctl b/shell-completion/zsh/_journalctl
index a469bbc9a..863348e05 100644
--- a/shell-completion/zsh/_journalctl
+++ b/shell-completion/zsh/_journalctl
@@ -76,6 +76,7 @@ _arguments -s \
     {-F,--field=}'[List all values a certain field takes]:Fields:_list_fields' \
     '--system[Show system and kernel messages]' \
     '--user[Show messages from user services]' \
+    {-M+,--machine=}'[Operate on local container]:machines:_sd_machines' \
     {-D+,--directory=}'[Show journal files from directory]:directories:_directories' \
     '--file=[Operate on specified journal files]:file:_files' \
     '--root=[Operate on catalog hierarchy under specified directory]:directories:_directories' \
